rmj_oss/server.py

```diff
@@ -68,23 +73,7 @@ def create_post():
     session.commit()
     # print('ID: ' + str(new_post.id)) # Prints this post's ID
 
-    return str(new_post.id)
```

Comment:
Is this removing the functionality to get a post by id? I don't see anything else to replace this.

Reply:
To get a post by ID. It has been moved to the search, because retrieving from the DB is categorized under that.